tests/unit/test_redfish.py

```diff
@@ -555,11 +565,69 @@ def mock_get_response(uri):
         mock_redfish_obj.get.assert_any_call("/redfish/v1/Systems/s1/Storage/STOR1")
         mock_redfish_obj.get.assert_any_call("/redfish/v1/Systems/s1/Storage/STOR2")
 
```

Comment:
Although these code are not written by you, I think it's unnecessary to check the API call url. Checking the expected output is good enough to me.

Reply:
IMO for `test_get_storage_controller_data_success`, we should be testing for the expected output and the correct redfish API call. It would be nice to test that we're calling the right endpoints with `redfish_obj.get` in order to fetch the correct results.

Reply:
Alright fair enough, I understand your point about the method being a black box and not having to test any implementation details. In that case we would need to modify almost all the test methods to remove the assertions for the API calls since the same point would apply to all of them and we will need to be consistent.

Reply:
In the same vein, we do not need to include the assertion below as well since we don't need to worry about whether the `redfish_utilities.systems.get_system_ids` method is called with `redfish_obj`. We could possibly obtain the system ids in other ways.

```python
mock_get_system_ids.assert_called_once_with(mock_redfish_obj)
```

I'll go ahead and remove this from the test methods as well.